### ISSUE TITLE
pyproj warning

### ISSUE THREAD

**Issue by jeweinberg** [Esri/arcgis-python-api#855]
looks like pyproj is causing an issue when trying to change projections. When I plot polygons after this error the polygons are broken with points out of order.

```python
data = pd.DataFrame.from_records([
    {'lat': 18.030293, 'lon': -66.594315},
    {'lat': 19.851411, 'lon': -155.089965}
])
sdf = pd.DataFrame.spatial.from_xy(data, 'lat', 'lon')
sdf.spatial.project(102100)
```
error:
```python
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:53: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  return _prepare_from_string(" ".join(pjargs))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:294: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  projstring = _prepare_from_string(" ".join((projstring, projkwargs)))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:53: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  return _prepare_from_string(" ".join(pjargs))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:294: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  projstring = _prepare_from_string(" ".join((projstring, projkwargs)))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:53: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  return _prepare_from_string(" ".join(pjargs))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:294: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  projstring = _prepare_from_string(" ".join((projstring, projkwargs)))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:53: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  return _prepare_from_string(" ".join(pjargs))
/home/INTERNAL/h14322/.conda/envs/geo_env/lib/python3.6/site-packages/pyproj/crs/crs.py:294: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  projstring = _prepare_from_string(" ".join((projstring, projkwargs)))
```

**Platform (please complete the following information):**
 - OS: ubuntu
 - Browser chrome
 - Python API Version 1.8.2

**Comment by achapkowski** [comment]
@jeweinberg 

Try this:
 ```sdf = pd.DataFrame.spatial.from_xy(data, 'lat', 'lon', 102100)```

**Comment by jeweinberg** [comment]
Ok, that could get me a solution for now. Will specifying sr in the from_xy method actually re-project the data, or does that just specify what projection the data is already in?

**Comment by achapkowski** [comment]
The `pyproj` messages are not errors, they are warnings, so it is reprojecting the data.  `pyproj` moved from v1 to v2 and on v2 they had breaking changes.

**Comment by jeweinberg** [comment]
Yeah, I see that it re-projects but the points of polygons go out of order. I think pyproj has an always_xy flag that needs to be turned to True to fix it.

**Comment by achapkowski** [comment]
@jeweinberg we have put in a fix for the 1.9 release.
